packages/terra-signature/tests/nightwatch/signature-spec.js

```diff
@@ -0,0 +1,20 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
Should we introduce tests to ensure that the drawing functionality is behaving as expected? I'm not sure how feasible that is though, but it'd be great to have tests to ensure the drawing behavior is working as expected.

Reply:
I'm not sure that there's currently a feasible way to test it, but I do agree it would be good to ensure that it is functioning correctly. I believe the ultimate solution for this is screenshot comparison, so maybe in the mean time we add a test that executes drawing functionality, but doesn't tests anything, so when screenshot comparison is added, we already have a test ready. Plus it may catch the odd javascript error in the mean time.